docs/release-notes/cloud/cloud-2023-02.md

```diff
@@ -0,0 +1,149 @@
+-   https://codacy.atlassian.net/browse/TS-249
+-   https://codacy.atlassian.net/browse/TS-242
+-   https://codacy.atlassian.net/browse/TS-233
+-   https://codacy.atlassian.net/browse/TS-224
+-   https://codacy.atlassian.net/browse/TS-206
+-   https://codacy.atlassian.net/browse/TS-148
+-   https://codacy.atlassian.net/browse/IO-424
+-   https://codacy.atlassian.net/browse/COV-188
+-   https://codacy.atlassian.net/browse/COV-184
+-   https://codacy.atlassian.net/browse/COV-181
+-   https://codacy.atlassian.net/browse/COV-147
+
+Jira issues with disabled release notes
+
+Bugs and Community Issues:
+-   https://codacy.atlassian.net/browse/TS-238
+-   https://codacy.atlassian.net/browse/TS-237
+-   https://codacy.atlassian.net/browse/TS-236
+-   https://codacy.atlassian.net/browse/TS-234
+-   https://codacy.atlassian.net/browse/TS-232
+-   https://codacy.atlassian.net/browse/TS-230
+-   https://codacy.atlassian.net/browse/TS-225
+-   https://codacy.atlassian.net/browse/TS-217
+-   https://codacy.atlassian.net/browse/TS-215
+-   https://codacy.atlassian.net/browse/TS-208
+-   https://codacy.atlassian.net/browse/TS-177
+-   https://codacy.atlassian.net/browse/TS-156
+-   https://codacy.atlassian.net/browse/TS-136
+-   https://codacy.atlassian.net/browse/TS-94
+-   https://codacy.atlassian.net/browse/TS-89
+-   https://codacy.atlassian.net/browse/TS-82
+-   https://codacy.atlassian.net/browse/TS-79
+-   https://codacy.atlassian.net/browse/TS-71
+-   https://codacy.atlassian.net/browse/TS-60
+-   https://codacy.atlassian.net/browse/TS-54
+-   https://codacy.atlassian.net/browse/TS-46
+-   https://codacy.atlassian.net/browse/TS-42
+-   https://codacy.atlassian.net/browse/TS-39
+-   https://codacy.atlassian.net/browse/TS-31
+-   https://codacy.atlassian.net/browse/IO-462
+-   https://codacy.atlassian.net/browse/IO-460
+-   https://codacy.atlassian.net/browse/IO-449
+-   https://codacy.atlassian.net/browse/IO-426
+-   https://codacy.atlassian.net/browse/IO-397
+-   https://codacy.atlassian.net/browse/IO-355
+-   https://codacy.atlassian.net/browse/IO-266
+-   https://codacy.atlassian.net/browse/IO-207
+-   https://codacy.atlassian.net/browse/IO-5
+-   https://codacy.atlassian.net/browse/COV-210
+-   https://codacy.atlassian.net/browse/COV-190
+-   https://codacy.atlassian.net/browse/COV-189
+-   https://codacy.atlassian.net/browse/COV-155
+-->
+
+## Product enhancements
+
+-   Improved the reporting of analysis and coverage status to Git providers when new commits are pushed during an ongoing analysis. (IO-341)
+-   Codacy now supports [reporting a summary of the coverage metrics](../../repositories-configure/integrations/github-integration.md#enabling) directly on your pull requests. For now, this feature is only supported on GitHub Cloud. ![Coverage summary on GitHub](../images/cov-2.png) (COV-2)
+
+## Bug fixes
+
+-   Fix email notification not sent when the user sending the request doesn't have a user name (PLUTO-365)
```

Comment:
I updated this release note in the [Jira issue](https://codacy.atlassian.net/browse/PLUTO-365):
"Fixed an issue that prevented organization owners from receiving the email notification with new requests to join the Codacy organization when the requester user didn’t have a name set up."